src/app/community-list-page/community-list-service.ts

```diff
@@ -139,6 +149,28 @@ export class CommunityListService {
         return buildPaginatedList(newPageInfo, newPage);
       })
     );
+
+    // TAMU Customization - process scoped collections
+    if (scopedCollections.length > 0) {
+      return observableCombineLatest(topComs$, observableCombineLatest([...scopedCollections])).pipe(
+        switchMap(([topComs, scopeCols]) => this.transformListOfCommunities(topComs, 0, null, expandedNodes).pipe(
+          map((nodes: FlatNode[]) => {
+            const colNodes = scopeCols.map(payload => {
+              let cNodes = payload.page.map((collection: Collection) => {
+                return toFlatNode(collection, observableOf(false), 0, false);
+              });
+              if (currentPage < payload.totalPages && currentPage === payload.currentPage) {
+                cNodes = [...cNodes, showMoreFlatNode(`collection-${uuidv4()}`, 0, null)];
+              }
+              return cNodes;
+            }).reduce((acc, e) => acc.concat(e), []);
+            return [...nodes, ...colNodes];
+          })
+        )),
+        // distinctUntilChanged((a: FlatNode[], b: FlatNode[]) => a.length === b.length)
```

Comment:
What is this comment here for?

Reply:
It came along with cut and paste. Someone else decided to leave it in and I just went with that choice as it will have to be uncommented in both places if they find it necessary. A not so defined scenario.

Reply:
I will remove it.

Reply:
From https://github.com/TAMULib/dspace-angular/pull/28/files#diff-7acc285319ea4672a90469f00b901697ba9e756ca4ed1970fbfceb99b24f7873R176

I can remove it.